splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -1595,6 +1595,9 @@
           "type": "string",
           "pattern": "^(?:\\d{1,3}\\.){2}\\d{1,3}$"
         },
+        "uccVersion": {
```

Comment:
What do you think if we make it `_uccVersion` to emphasize that it is an internal thing?